docs/learn/expert-guides/universal-profile/extend-profile-functionalities.md

```diff
@@ -0,0 +1,556 @@
+---
+sidebar_label: '⚙️ Extend a Universal Profile Functionalities'
+sidebar_position: 4
+---
```

Comment:
Could you please add a SEO friendly description?

https://github.com/lukso-network/docs/pull/843

```suggestion
sidebar_position: 4
description: XXXX (max 150char) for SEO
---
```